test/k8sT/Policies.go

```diff
@@ -854,9 +854,10 @@ var _ = Describe("K8sPolicyTest", func() {
 
 			checkProxyRedirection := func(resource string, redirected bool, parser policy.L7ParserType) {
 				var (
-					not     = " "
-					reStr   string
-					curlCmd string
+					not                  = " "
+					reStr                string
+					curlCmd              string
+					monitorOutputTimeout = 10 * time.Second
```

Comment:
Reviewers: We can use helpers.ShortCommandTimeout here but that seemed unclear to me so I did this. I can change it.